sentry/src/lib.rs

```diff
@@ -71,8 +82,13 @@ where
     }
 }
 
-async fn handle_routing(req: Request<Body>, adapter: impl Adapter) -> Response<Body> {
-    if req.uri().path().starts_with("/channel") {
+async fn handle_routing(
+    req: Request<Body>,
+    (adapter, config): (impl Adapter, Config),
+) -> Response<Body> {
+    if req.uri().path().starts_with("/cfg") && req.method() == Method::GET {
```

Comment:
Can we use `match` statement here instead of `if else` ?

Reply:
We could try, but I rather leave it as simple as possible right now. we need to do something like:
```
match (req.uri().path(), req.method()) {
   (string, method) if string.starts_with("/cfg") && method == Method::GET => ....,
   (string) if string.starts_with("/channel") => ....
}
```

way too much complications for now + if we decide we need Regex for future route we need to probably revert back to `if-else`s

Reply:
I don't think it's too much complications, it's much better.
Also, thinking about it I don't think we should be using `starts_with` it should be equilvalent string match i.e `path() == "/cfg"` not starts_with.
```
match (req.uri().path(), req.method()) {
   ("/cfg", Method::GET) => ()..,
```
Also on Regex we would be using `RegexSet` https://docs.rs/regex/1.3.1/regex/#example-match-multiple-regular-expressions-simultaneously. And a match statement should fit in okay there too

Reply:
Done.